content/blog/posts/o3de-ux-past-present-future.md

```diff
@@ -0,0 +1,95 @@
+
+At this point, I worked with the UX team to redesign the Project Manager and Gem Catalog. For those who are new to this, Gems are packages of code or assets that augment game or simulation projects, and the Gem Catalog is a library of Gems that are native to O3DE. We conducted user interviews to understand the overall experience of the Gem system, any suggestions and issues, and the frequency with which users configure their project with Gems.
+
+>"_One of our biggest problems is not knowing what Gems are given to us, when installing or upgrading the application._"&mdash;**O3DE Research Participant**
+
+We found that it was difficult for new users to wrap their mind around the concept of Gems and their use within projects. With 80 Gems native to O3DE, the typical usage varies over two patterns—users either retain the default configuration throughout their development process or they enable all Gems to shorten build and compilation time. It was important to establish an information hierarchy, because of the Gems' low ==findability== and ==discoverability==.
+
+**Findability**: The ease with which information can be found within a system.
+
+**Discoverability**: The degree to which information can be found within a system.
+
+The fundamental objective for the system's architecture was to allow users to be aware of where they are, what they found, what is around, and what to expect. As a result, we conducted a card sort study where participants were asked to arrange a stack of virtual cards into distinct categories based on their understanding of Gems. We identified the key category for the faceted navigation to be *Features*, which would classify Gems on the basis of their distinctive attributes and functionalities.
+
+![O3DE UX card sort image](/images/blog/bjds-posts/04-card-sort-session.gif)
+
+With the research findings in mind, we established a set of design principles. Our goal was to identify a design solution that aligns with these principles while addressing the needs of all users.
+
+**Discover, Find Search**
+Users must be able to explore the Gem Catalog efficiently. We improved the searchability of the catalog with filter, sort, and search functions. This allows users to make appropriate decisions for their project during Gem selection.
+
+**Reinforce Clarity to Foster Learning**
+The design must give users constant feedback to help them recognise and recover from errors. By providing help and documentation at all steps, and by creating a clear informational hierarchy we can help reduce error and encourage user confidence.
+
+**Autonomy to Encourage Contribution and Maintain Consistency and Standards**
```

Comment:
`**Autonomy to Encourage Contribution and Maintain Consistency and Standards:**<br>`